c2corg_api/emails/email_service.py

```diff
@@ -1,11 +1,33 @@
 from pyramid_mailer import get_mailer
 from pyramid_mailer.message import Message
+from functools import lru_cache
+from c2corg_common.attributes import default_langs
 
 import logging
+import os
 
 log = logging.getLogger(__name__)
 
 
+class EmailLocalizator(object):
+    def __init__(self):
+        self.default_lang = 'fr'
+
+    @lru_cache(maxsize=None)
+    def _get_file_content(self, lang, key):
+        filepath = os.path.dirname(__file__) + '/i18n/%s/%s' % (lang, key)
```

Comment:
Do you maybe want to use [os.path.isfile(path)](https://docs.python.org/2/library/os.path.html#os.path.isfile) to check first if the file exists?